code/index.html

```diff
@@ -4,27 +4,98 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
+    
+    <!-- </header> -->
     <!-- video or image as a header is cool :) -->
+<header class="hero">
+  <video autoplay control playsinline loop muted poster="./img/sandy-millar-qRB6rdYQ_8Q-unsplash.jpg">
+    <source src="./videos/SampleVideo_360x240_1mb.mp4" type="video/mp4">
+    <source src="./videos/SampleVideo_360x240_1mb.webm" type="video/webm">
+    <source src="./videos/SampleVideo_720x480_1mb.mp4" type="video/mp4">
+    <source src="./videos/SampleVideo_720x480_1mb.webm" type="video/webm">
+    <source src="./videos/SampleVideo_1280x720_1mb.mp4" type="video/mp4">
+    <source src="./videos/SampleVideo_1280x720_1mb.webm" type="webm">
+    <p>This browser dosen't support</p>
+  </video>
+  <h1 class="hero-h1">GOOD MORNING!</h1>
+</header>
 
-    <!-- Signup form -->
+<!-- Text-section -->
+<section class="text-wrapper">
+  <h2>Do you love to wake up and start a new day?</h2>
+  <h2>Is coffe life? </h2>
+  <h2>Come and join our community today!</h2>
+</section>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
+
+
+<!-- Signup form -->
+<section class="content-wrapper">    
+  <Div class="form-wrapper">
+    <form action="https://httpbin.org/anything" method="post">
+      <h1>Sign up now</h1> 
+        <label>
+          <input type="text" name="name" id="name" placeholder="Name">
```

Comment:
Could try to add 'required' tag in input to prevent the form from submitting empty of content. The form works fine as it is now but this could improve user interaction :)